macos/package_files/3.9.3/postinstall.sh

```diff
@@ -14,6 +14,12 @@ DIR="/Library/Ossec"
 INSTALLATION_SCRIPTS_DIR="${DIR}/packages_files/agent_installation_scripts"
 SCA_FILES_DIR="${INSTALLATION_SCRIPTS_DIR}/sca"
 
+if [ -d $DIR/config_files ]; then
+    rm -rf /Library/Ossec/etc/{ossec.conf,client.keys,local_internal_options.conf,shared}
```

Comment:
Don't hardcode the path `/Library/Ossec`. There's a variable for this.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/c1c92ee5126c339b322e7fb1fa03b2c422dacbf1